packages/os/os.spec

```
@@ -423,6 +424,13 @@ Conflicts: %{_cross_os}bash
 %description -n %{_cross_os}brush
 %{summary}.
 
+%package -n %{_cross_os}whippet
+Summary: A simple dbus-broker launcher
+Provides: %{_cross_os}dbus-launcher = %{package_priority_epoch}:
+Conflicts: %{_cross_os}dbus-launcher
```

I'd recommend using parentheses for the capabilities that aren't expected to match a package name:
```suggestion
Provides: %{_cross_os}dbus-broker(launcher) = %{package_priority_epoch}:
Conflicts: %{_cross_os}dbus-broker(launcher)
```